api/core/v2/apikey.go

```diff
@@ -0,0 +1,70 @@
+package v2
+
+import (
+	"fmt"
+	"net/url"
+	"path"
+)
+
+const (
+	// APIKeysResource is the name of this resource type.
+	APIKeysResource = "apikeys"
```

Comment:
Do you think this should be `api-keys`? I did `etcd-replicators` recently, I feel like it reads a bit better to have a dash.

Reply:
Readability vs consistency, it's a great question. To me `/clusterrolebindings` is clunky, but it's the naming convention we went with for corev2. Happy to discuss further if we want to break the convention.